commons/connector/fitbit/fitbit_food_log.avsc

```diff
@@ -0,0 +1,108 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitFoodLog",
+  "doc": "A summary of the user's food log entry for a given day.",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)."
+    },
+    {
+      "name": "isFavorite",
+      "type": "boolean",
+      "doc": "Whether the food is the favorite one."
+    },
+    {
+      "name": "logId",
+      "type": "long",
+      "doc": "The id of the logged food."
+    },
+    {
+      "name": "accessLevel",
+      "type": "string",
+      "doc": "The access level of the food."
+    },
+    {
+      "name": "amount",
+      "type": "int",
+      "doc": "The amount of the food."
+    },
+    {
+      "name": "brand",
+      "type": "string",
+      "doc": "The brand of the food."
+    },
+    {
+      "name": "foodId",
+      "type": "long",
+      "doc": "The id of the food."
+    },
+    {
+      "name": "locale",
+      "type": "string",
+      "doc": "The locale of the food."
+    },
+    {
+      "name": "mealTypeId",
+      "type": "long",
+      "doc": "The meal type id of the food."
+    },
+    {
+      "name": "name",
+      "type": "string",
+      "doc": "The name of the food."
+    },
+    {
+      "name": "unitId",
+      "type": "long",
+      "doc": "The unit id of the food."
+    },
+    {
+      "name": "unitName",
+      "type": "string",
+      "doc": "The unit name of the food."
+    },
+    {
+      "name": "plural",
+      "type": "string",
+      "doc": "The plural of the food."
+    },
+    {
+      "name": "calories",
```

Comment:
Just to be sure: is this calories or kilocalories?

Reply:
I am not sure. Nothing specified in the description here -- https://dev.fitbit.com/build/reference/web-api/nutrition/get-food-log/
But looking at the example response it seems like it would be Kilocalories